### PR TITLE
[locale] ko: Update ordinal format

### PR DESCRIPTION
**Pull request by FourwingsY**
almost copied from zh-cn.js

### CODE REVIEW

File: locale/ko.js [hunk 1]
```diff
@@ -49,8 +49,22 @@ var ko = moment.defineLocale('ko', {
         y : '일 년',
         yy : '%d년'
     },
-    ordinalParse : /\d{1,2}일/,
```
Comment (fbonzon) [inline locale/ko.js]: Please rebase from the HEAD of the develop branch. This was renamed `dayOfMonthOrdinalParse` 5 days ago.

**Comment by FourwingsY** [comment]
@fbonzon sorry for late check. i've updated it.

**Comment by ichernev** [comment]
@FourwingsY  you need to update `src/locale/ko.js` and `src/test/locale/ko.js`.

**Comment by FourwingsY** [comment]
@ichernev done.

**Comment by ichernev** [comment]
@kyungw00k what do you think?

**Comment by ichernev** [comment]
@FourwingsY can you also undo the changes to `locale/ko.js`, its autogenerated on release.

**Comment by kyungw00k** [comment]
@ichernev 👌 
@FourwingsY 👍

**Comment by FourwingsY** [comment]
yup

**Comment by ichernev** [comment]
Merged in eef541ec2707613a200ab6dc0dc14c314785d9ef

**Pull request closed without merging**